packages/admin/src/api.ts

```diff
@@ -66,10 +73,73 @@ export class AdminClient extends BaseApiClient {
     });
   }
 
-  public async createProposal(proposal: CreateProposalRequest): Promise<ProposalResponseWithUrl> {
+  // added separate from CreateProposalRequest type as the `simulate` boolean is contained within defender-client
+  public async createProposal(
```

Comment:
createProposal has gotten much more complicated I am wondering is a createSimulateProposal would not be more ideal?

Reply:
I don't think that makes sense in this case, see https://github.com/OpenZeppelin/defender-client/pull/183#discussion_r1036233644

Reply:
True!